File: Domain/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class Player
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Position { get; set; } // public Positions Position { get; set; }
        public int Goals { get; set; }
        public Guid TeamId { get; set; }
        public Team Team { get; set; }

        // public enum Positons
        // {
        //     Bramkarz,
        //     Obrońca,
        //     Pomocnik,
        //     Napastnik
        // }
    }
}

File: ui/src/app/models/Player.ts
export interface Player {
    id: string;
    name: string;
    position: string;
    goals: number;
    teamId: string;
  }
  
  export class Player implements Player {
    constructor(init?: PlayerDto) {
      Object.assign(this, init);
    }
  }
  
  export class PlayerDto {
    id?: string = undefined;
  
    constructor(player?: PlayerDto) {
      if (player) {
        this.id = player.id;
      }
    }
  }